### PR TITLE
Check if admin user

### PR DESCRIPTION
**Pull request by LukasReschke**
We should not add the JS for regular users.

@nickvergessen THX for noticing.
@MorrisJobke Fancy a review?

### CODE REVIEW

**Comment by nickvergessen** [comment]
Removes the link that I see but can not follow on my update test.
:+1:

**Comment by LukasReschke** [comment]
To test this run the update to 9.0 with a failed integrity check (e.g. add a new file into the folder) and then login as user. The expected behaviour is that only admin see this error message. The check got missing at some point :see_no_evil:

File: lib/private/templatelayout.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ public function __construct( $renderAs, $appId = '' ) {
 
 			// Code integrity notification
 			$integrityChecker = \OC::$server->getIntegrityCodeChecker();
-			if(!$integrityChecker->hasPassedCheck()) {
+			if(\OC_User::isAdminUser(\OC_User::getUserSession()) && !$integrityChecker->hasPassedCheck()) {
```
Comment (MorrisJobke) [inline lib/private/templatelayout.php]: Should be getUser() - I will push a fix.

**Comment by MorrisJobke** [comment]
I tested this and now it shows properly the warning for the admin and no warning for a normal user. :+1:

**Comment by rullzer** [comment]
:+1:

**Pull request merged**